api/diagnostics.go

```diff
@@ -485,12 +485,11 @@ func (d diagnosticsJobCanceledError) Error() string {
 // fetch an HTTP endpoint and append the output to a zip file.
 func (j *DiagnosticsJob) getHTTPAddToZip(node Node, endpoints map[string]string, zipWriter *zip.Writer,
 	summaryErrorsReport, summaryReport *bytes.Buffer, percentPerNode float32) error {
-	if len(endpoints) == 0 || percentPerNode == 0 {
```

Comment:
just wondering why these checks can be removed? (we don't need to worry about a divide by 0 panic?)

Reply:
It's covered here https://github.com/dcos/dcos-diagnostics/pull/60/files#diff-0ee9de7b9d9927cce9fb53251662d72aR303 (Github fold this part).

Reply:
I mocked `/logs` endpoint in one tests to validate this case.